### PR TITLE
debs/xenial: change venv install root

### PR DESCRIPTION
**Pull request by sevein**
Move virtual environments to `/usr/share/archivematica/virtualenvs/*`.

This relates to #119. This PR applies to debs/xenial, if Santi is ok with the approach I will submit another PR extending the solution to debs/trusty and rpms for consistency.

### CODE REVIEW

**Comment by sevein** [comment]
@scollazo can you please advise on what would be the best way to test these changes? My `rpm-testing` lets me test a local repo easily but I don't know how to do the same here in `debs/xenial`.

File: debs/xenial/archivematica-storage-service/debian-storage-service/postrm [hunk 1]
```diff
@@ -7,6 +7,7 @@ echo "postrm" $1
 if [ $1 = "remove" ]; then
     #clean up storage service directories
     rm -rf /usr/share/python/archivematica-storage-service
```
Comment (sevein) [inline debs/xenial/archivematica-storage-service/debian-storage-service/postrm]: Should I remove this line, @scollazo? I wasn't entirely sure.

**Comment by sevein** [comment]
I've tested this and it seems to be working. Please review!

File: debs/xenial/archivematica-storage-service/debian-storage-service/postrm (hunk 1 above)
Comment (scollazo) [reply debs/xenial/archivematica-storage-service/debian-storage-service/postrm]: Cleaning files after removing the package  doesn't hurt, and avoids piling up space used after uninstalling the package

**Comment by sevein** [comment]
> Did you test an upgrade from 1.6?

I just did and it looks like the removal of the old venvs need to happen in advance so I filed https://github.com/artefactual/archivematica-docs/issues/100 to update the docs.

**Pull request merged**